data_hub_api/docmaps/v2/api_input_typing.py

```diff
@@ -40,7 +40,8 @@
         'preprint_version': str,
         'preprint_published_at_date': date,
         'tdm_path': str,
-        'evaluations': Sequence[ApiEvaluationInput]
+        'evaluations': Sequence[ApiEvaluationInput],
+        'manuscript_published_date': str
```

Comment:
Should this not be an optional  `date` field?

Reply:
Perhaps as a separate PR we could also review whether there are other optional fields, e.g. `tdm_path`?